content/en/platform/corda/4.10/enterprise/get-started/getting-started-with-corda-4.md

```diff
@@ -20,11 +21,12 @@ The best way to get started with Corda is to:
      * Download the latest Corda Community Edition `.tar` [file](https://download.corda.net/corda-community-edition/4.10/community-4.10.tar) that contains the required Corda `.jars`. 
      * Use the Docker image and accompanying guide from the [Docker Hub](https://hub.docker.com/repository/docker/corda/community).
 
-2. Familiarize yourself with the [Corda key concepts](./corda-key-concepts.md). 
-3. Run a [sample CorDapp](../../en/platform/corda/4.10/community/tutorial-cordapp.md) to see Corda in action.
-4. [Build your own CorDapp](../../en/tutorials/corda/4.10/community/build-basic-cordapp/basic-cordapp-intro.md) from scratch.
+2. Familiarize yourself with the [Corda key concepts](../../enterprise/about-corda/corda-key-concepts.md). 
+3. Run a [sample CorDapp](../../community/tutorial-cordapp.md) to see Corda in action. 
+(../../community/key-concepts-time-windows.md)
+4. [Build your own CorDapp](../../community/building-a-cordapp-index.md) from scratch.
 
-You can then take your Corda integration to the next level by deep-diving into specific topics in our [Corda Community Edition](../../en/platform/corda/4.10/community.html) documentation, or [Corda Enterprise](../../en/platform/corda/4.10/enterprise.html) documentation. You can also view the [API reference material](../../en/api-ref.html), browse [featured apps](../../en/apps.html), or browse [tools and add-ons](../../en/tools.html).
+You can then take your Corda integration to the next level by deep-diving into specific topics in our [Corda Enterprise](../../enterprise/_index.md) documentation. You can also view the [API reference material](../../../../../api-ref/_index.md), browse [featured apps](../../../../../apps/_index.md), or browse [tools and add-ons](../../../../../tools/_index.md).
```

Comment:
These links all seem to be broken in your preview site.

Reply:
caching